digiwf-apps/packages/apps/digiwf-webcomponent/vite.config.ts

```diff
@@ -0,0 +1,61 @@
+import { fileURLToPath, URL } from "node:url";
+
+import { viteVueCESubStyle } from "@unplugin-vue-ce/sub-style";
+import vue from "@vitejs/plugin-vue";
+import { defineConfig, loadEnv, PluginOption } from "vite";
+import cssInjectedByJsPlugin from "vite-plugin-css-injected-by-js";
+
+const portFromDevelopmentEnv = loadEnv("development", "./")?.VITE_PORT;
+const port = portFromDevelopmentEnv
+  ? Number.parseInt(portFromDevelopmentEnv)
+  : 8081;
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [
+    vue({
+      customElement: true,
+    }),
+    viteVueCESubStyle() as PluginOption,
+    cssInjectedByJsPlugin(),
+  ],
+  server: {
+    port,
+    proxy: {
+      "/api": "http://localhost:8083/",
+      "/actuator": "http://localhost:8083/",
+      "/clients": "http://localhost:8083/",
+    },
+  },
+  resolve: {
+    alias: {
+      "@": fileURLToPath(new URL("./src", import.meta.url)),
+    },
+    extensions: [".js", ".json", ".jsx", ".mjs", ".ts", ".tsx", ".vue"],
+  },
+  build: {
+    ssrManifest: true,
+    manifest: true,
+    minify: true,
+    outDir: "dist",
+    emptyOutDir: false,
+    rollupOptions: {
+      input: {
+        "digiwf-service-instances-webcomponent":
+          "./src/digiwf-service-instances-webcomponent.ts",
+        "digiwf-hello-world-webcomponent":
+          "./src/digiwf-hello-world-webcomponent.ts",
```

Comment:
Soll ich diese Dummy-WebComp noch entfernen? Ist eigentlich nur dafür gedacht gewesen, den Build mehrerer WebComponents in separate Dateien zu testen.

Reply:
Wenn ichs richtig gesehen habe sind das ja nur zwei Dateien, die kannst von mir aus dann gerne drin lassen.

Reply:
Mit @simonhir besprochen, dass die Komponente drin behalten wird.

Reply:
Also ich würde sie entweder rauswerfen oder zumindest eine Mini-Funktion draus machen, z.B. Open-DigiWF-Button oder so. Als Hello World wird das Ding auf kurz oder lang nur unnötiger Code sein, den keiner warten möchte.